### PR TITLE
add script for a dataset using logjuicer

### PR DESCRIPTION
**Pull request by TomasTomecek**
~~requires HEAD of logjuicer~~

requires logjuicer 0.9.10 :)

### CODE REVIEW

File: backend/src/fetcher.py [hunk 1]
```diff
@@ -160,14 +160,27 @@ def fetch_spec_file(self) -> Optional[dict[str, str]]:
         return {"name": spec_name, "content": response.text}
 
 
+def find_matching_koji_subtask(
```
Comment (jpodivin) [inline backend/src/fetcher.py:163]: I'm not sure I understand how is this related to the standalone scripts this PR is introducing?

Comment (TomasTomecek) [reply backend/src/fetcher.py:163]: I reused the backend logic in the new script: https://github.com/fedora-copr/log-detective-website/pull/114/files#diff-816f1b55c694752aa1794eb4c879b417609a56727a3b902d0b11b2960ea3a4ecR19

**Comment by TomasTomecek** [comment]
Closing as this is no longer relevant.

We may revisit logjuicer in the future.
